hilgendorf_support/launch/driver_hilgendorf.launch

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!--
+  Slightly modified Universal robot common bringup.  Starts ur driver node and robot state
+  publisher (translates joint positions to propper tfs). Adds extra argument for prefix.
+  
+  NOTE: This launch file also starts a tf2 node, which is not typically
+  used
```

Comment:
Copy/pasta? Seems not this file, but `robot_interface_streaming_hilgendorf.launch` and `robot_state_visualize_hilgendorf.launch` do that.